scripts/update-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
Probably it'd be a good idea to use similar script in other repos at some point. We might also tweak it to update items which are updated very rarely, e.g. `Copyright © 2019 Mapbox, Inc.`.

Reply:
Oh gosh, that’s a great idea. 🙈

https://github.com/mapbox/mapbox-navigation-ios/blob/fb5e92d51961ddf08acf46c5169b553f6d52dcb5/docs/jazzy.yml#L7